lib/rubycritic/platforms/linux.rb

```diff
@@ -4,14 +4,10 @@ module Rubycritic
   module Platforms
     class Linux < Base
       BROWSER_MAP = {}.tap do |hash|
-        hash[:chrome] = 'google-chrome'
+        hash[:chrome]   = 'google-chrome'
         hash[:chromium] = 'chromium-browser'
-        hash[:firefox] = 'firefox'
-        hash.default = 'firefox'
-      end
-
-      def can_open?
-        platform == :linux || platform == :unix
+        hash[:firefox]  = 'firefox'
+        hash.default    = 'firefox'
```

Comment:
That'd be great, but it can be quite a lot of work.
At least for debian-based Linux distros the `sensible-browser` command should be available for us to use. But I don't know about the other platforms.